packages/lab/src/date-input/DateInputRange.tsx

```diff
@@ -214,7 +214,7 @@ export const DateInputRange = forwardRef<
       endInputRef: endInputRefProp,
       locale,
       parse: parseProp,
-      placeholder = "dd mmm yyyy",
+      placeholder = format,
```

Comment:
nice

Reply:
Casing actually matters, in certain library, lower case and upper case means different things.